packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/footer/disableAdd.ts

```diff
@@ -12,16 +12,26 @@ export const disableAdd = (
   switch (currWidgetType) {
     case 'status':
     case 'kpi': {
-      const assets =
+      const modeledProps =
         get(selectedWidget, 'properties.queryConfig.query.assets') ?? [];
-      if (assets.length) {
+      const unmodeledProps =
+        get(selectedWidget, 'properties.queryConfig.query.properties') ?? [];
+      const assetModelProps =
+        get(selectedWidget, 'properties.queryConfig.query.assetModels') ?? [];
```

Comment:
Could you change these names to `*Properties` instead of `*Props`. Props is an overloaded term and it adds confusion, especially since there is `collectionProps` in the same function.